src/wallet/wallet.cpp

```diff
@@ -1332,12 +1329,12 @@ bool CWallet::AbandonTransaction(interfaces::Chain::Lock& locked_chain, const ui
     return true;
 }
 
-void CWallet::MarkConflicted(const uint256& hashBlock, const uint256& hashTx)
+void CWallet::MarkConflicted(const uint256& hashBlock, int conflicting_height, const uint256& hashTx)
 {
     auto locked_chain = chain().lock();
     LOCK(cs_wallet);
 
-    int conflictconfirms = -locked_chain->getBlockDepth(hashBlock);
+    int conflictconfirms = -conflicting_height;
```

Comment:
I think this is a change in behaviour. The was previously (current tip height) - (height of block with conflicting tx). This is now just (height of block with conflicting tx).

Reply:
Thanks for this one, in fact after thinking of it, that would be a bug if you have multiple conflicting txn, you would pick up the one with the highest height instead of the highest depth so not the most conflicting one..